homework_7/functional.py

```diff
@@ -0,0 +1,72 @@
+import sys
+
+def sequential_map(*args):
+    """Takes several functions and a container and returns a list 
+    of all container values after sequentional applying of functions to them"""
+    funcs = args[:-1]
+    container = args[-1]
```

Comment:
Мне нравится вот так
```suggestion
    *funcs, container = args
```